crates/wit-component/tests/components/adapt-preview1/module.wat

```diff
@@ -3,8 +3,8 @@
   (import "foo" "foo" (func))
 
   ;; import some wasi functions
-  (import "wasi_snapshot_preview1" "proc_exit" (func (param i32)))
-  (import "wasi_snapshot_preview1" "random_get" (func (param i32 i32) (result i32)))
+  (import "wasi-snapshot-preview1" "proc_exit" (func (param i32)))
```

Comment:
Yeah I tried to figure out a good way here but I concluded in the end it wasn't worth it. This doesn't really have anything to do with WASI itself but moreso the conventions of the tests here where `foo.wat` has a corresponding `foo.wit` and the `foo` part of the filestem has to be a valid WIT identifier because it becomes the document name.

Currently I haven't added various APIs to `UnresolvedPackage::parse` to handle a document/package name different from the filename itself, so while that could be done I figured it may not be too worth it as this is just a test-convention thing rather than a problem-with-WASI thing.